### ISSUE TITLE
Artens status - mangler noe logikk

### ISSUE THREAD

**Issue by AneMarlene** [Artsdatabanken/Fremmedartsbase2023#460]
Svarer de "ja" på denne, er konklusjonen at arten ikke skal risikovurderes, og arten får risikokategori NR. Ingen flere spørsmål stilles på fanen:

<img width="386" alt="image" src="https://user-images.githubusercontent.com/58254444/153753647-b1e17351-8f79-4985-aaea-a12252686bfc.png">

**Comment by steinho** [comment]
Hvis de svarer ja på at taksonet vurderes på høyere nivå?

**Comment by AneMarlene** [comment]
Ja, hvis de svarer ja på høyere (artsnivå).... , så skal det bli NR osv osv :)

**Comment by steinho** [comment]
Er dette snakk om to ting som skal endres:

Kategori blir i dag ikke NR i dette tilfellet? At det må fikses?
De neste spørsmålene skal skjules hvis ja på spm?

**Comment by steinho** [comment]
Den første er kanskje en Helge oppgave - det er han som beregner riktig kategori og kriterier...
